app/src/main/java/com/arygm/quickfix/ui/search/SearchOnBoarding.kt

```diff
@@ -105,55 +122,87 @@ fun SearchOnBoarding(
                           textColor = colorScheme.onBackground,
                           placeHolderColor = colorScheme.onBackground,
                           leadIconColor = colorScheme.onBackground,
-                          widthField = 320.dp * widthRatio.value,
-                          heightField = 40.dp,
-                          moveContentHorizontal = 10.dp,
+                          widthField = 300.dp * widthRatio,
+                          heightField = 40.dp * heightRatio,
+                          moveContentHorizontal = 10.dp * widthRatio,
                           moveContentBottom = 0.dp,
                           moveContentTop = 0.dp,
-                          sizeIconGroup = 30.dp,
+                          sizeIconGroup = 30.dp * sizeRatio,
                           spaceBetweenLeadIconText = 0.dp,
                           onClick = true,
                       )
-                      Spacer(modifier = Modifier.width(10.dp))
+                      Spacer(modifier = Modifier.width(10.dp * widthRatio))
                       QuickFixButton(
                           buttonText = "Cancel",
-                          textColor = colorScheme.onSecondaryContainer,
+                          textColor = colorScheme.onBackground,
                           buttonColor = colorScheme.background,
                           buttonOpacity = 1f,
-                          textStyle = poppinsTypography.labelSmall,
+                          textStyle = poppinsTypography.bodyMedium,
```

Comment:
I don't understand theses changes here, it doesn't match the Figma at all, I think it's better if you keep it the way it was since now the Cancel button is not event vertically centered with the search componenent :/

Reply:
I can't remember the reason for it. I changed it back to the `poppinsTypography.labelSmall` !